code/modules/cargo/expressconsole.dm

```diff
@@ -55,9 +55,9 @@
 			continue// by using someone else's crate
 		if(!(obj_flags & EMAGGED) && P.contraband) // will you show me?
 			continue // i'd be right happy to
-		meme_pack_data[P.group]["packs"] += list(list(
+		meme_pack_data[P.group]["packs"] += list(list(//mfw goof sullies your nice cargo features with n a s t y and downright s i l l y comments^^^
 			"name" = P.name,
-			"cost" = P.cost * 2, //displays twice the normal cost
+			"cost" = P.cost * 1.2, //displays 0.2x normal cost

```

Comment:
um it's not marked down 80%.

Just remove the comment or replace it with the rationale for having it a slightly higher cost so people don't try to make a "consistency" pr in the future.

Reply:
mfw im retard